python/mathsys/dev/__init__.py

```
@@ -87,4 +102,4 @@ def wrapper(*arguments: str) -> None:
             components.pop()
             with open(".".join(components), "wb") as destination:
                 destination.write(unix_x86_64(content))
-        case _: sys.exit(f"[ENTRY ISSUE] Unknown command. Available commands: watch, validate, latex, web, unix-x86-64.")
\ No newline at end of file
+        case _: sys.exit(f"[ENTRY ISSUE] Unknown command. Available commands: validate, latex, web, unix-x86-64.")
```

Ruff (0.14.4)

Multiple statements on one line (colon)

(E701)